modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -150,6 +150,7 @@ function(trusted_firmware_build)
                ${TFM_PROFILE_ARG}
                ${MCUBOOT_IMAGE_NUM_ARG}
                -DTFM_TEST_REPO_PATH=${ZEPHYR_TRUSTED_FIRMWARE_M_MODULE_DIR}/tf-m-tests
+               -DMBEDCRYPTO_PATH=${ZEPHYR_TRUSTED_FIRMWARE_M_MODULE_DIR}/../tfm-mbedcrypto
```

Comment:
This may collide with an externally set -DMBEDCRYPTO_PATH
We set this already for nrf_security in nRF Connect SDK

Could this be handled by a specific variable instead of using a relative/full pass so downstream users can inject their own settings?